boards/arm/nrf5340_audio_dk_nrf5340/nrf5340_audio_dk_nrf5340_cpuapp_common-pinctrl.dtsi

```diff
@@ -66,6 +66,24 @@
 		};
 	};
 
+	pwm0_default: pwm0_default {
+		group1 {
+			psels = <NRF_PSEL(PWM_OUT0, 0, 7)>,
+				<NRF_PSEL(PWM_OUT1, 0, 25)>,
+				<NRF_PSEL(PWM_OUT2, 0, 26)>;
+			nordic,invert;
```

Comment:
```suggestion
```
The PWM is to use normal polarity for those outputs (as the LEDs are active high), so this property needs to be removed.

Reply:
done